src/index.tsx

```diff
@@ -2,11 +2,12 @@ import './index.css';
 import App from './App';
 
 import React from 'react';
-import ReactDOM from 'react-dom';
+import {createRoot} from 'react-dom/client';
```

Comment:
The rendering API changed in React 18
https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-client-rendering-apis